gnucash/report/standard-reports/transaction.scm

```diff
@@ -417,7 +417,12 @@ tags within description, notes or memo. ")
                   (N_ "Include transactions to/from filter accounts only."))
           (vector 'exclude
                   (N_ "Exclude Transactions to/from Filter Accounts")
-                  (N_ "Exclude transactions to/from all filter accounts.")))))
+                  (N_ "Exclude transactions to/from all filter accounts.")))
+   #f
+   (lambda (x)
+     (gnc-option-db-set-option-selectable-by-name
+      options gnc:pagename-accounts optname-filterby
+      (not (eq? x 'none))))))
```

Comment:
Your commit message doesn't explain this very well. And while on that subject, it looks like this commit is a mixture of at least 2 changes that are not related - a comment cleanup and a change in the info box functionality. These beg for separate commits. Another opportunity for git history rewriting...